charts/bindplane/templates/bindplane.yaml

```
@@ -146,6 +146,46 @@ spec:
               value: /credentials.json
             {{- end }}
             {{- end }}
+            {{- if and (eq .Values.enterprise true) (eq .Values.eventbus.type "kafka") }}
+            - name:  BINDPLANE_KAFKA_BROKERS
+              value: {{ .Values.eventbus.kafka.brokers }}
+            - name:  BINDPLANE_KAFKA_PROTOCOL_VERSION
+              value: "{{ .Values.eventbus.kafka.protoclVersion }}"
```

Misspelled `protocol`
```suggestion
              value: "{{ .Values.eventbus.kafka.protocolVersion }}"
```